src/badguy/badguy.cpp

```diff
@@ -905,4 +905,24 @@ BadGuy::after_editor_set()
   }
 }
 
+bool
+BadGuy::can_be_affected_by_wind() const
+{
+  return !on_ground();
+}
+
+void
+BadGuy::add_wind_velocity(const Vector& velocity, const Vector& end_speed)
+{
+  // only add velocity in the same direction as the wind
```

Comment:
I missed the superfluous space around the `&&`here as well. If you don't mind, please fix this as well, then it can get merged.

Reply:
Have fixed now.